x-pack/plugins/aiops/public/hooks/use_filters_query.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type FC, createContext, useEffect, useState, useContext } from 'react';
+import type { Filter, Query, TimeRange } from '@kbn/es-query';
+import { useTimeRangeUpdates } from '@kbn/ml-date-picker';
+import { type AggregateQuery } from '@kbn/es-query';
+import { useAiopsAppContext } from './use_aiops_app_context';
+
+export const FilterQueryContext = createContext<{
+  filters: Filter[];
+  query: Query;
+  timeRange: TimeRange;
+}>({
+  get filters(): Filter[] {
+    throw new Error('FilterQueryContext is not initialized');
+  },
+  get query(): Query {
+    throw new Error('FilterQueryContext is not initialized');
+  },
+  get timeRange(): TimeRange {
+    throw new Error('FilterQueryContext is not initialized');
+  },
+});
+
+/**
+ * Helper context to provide the latest filter, query and time range values
+ * from the data plugin.
+ * Also merges custom filers and queries provided with an input.
```

Comment:
Nit - `filters`

Reply:
Fixed in https://github.com/elastic/kibana/pull/162796/commits/29572a9a01dda92122520f4dd34e8f7318d8d552